test/Microsoft.AspNet.Razor.Test/Parser/CSharp/CSharpSectionTest.cs

```diff
@@ -170,8 +170,30 @@ public void ParseSectionBlockHandlesEOFAfterOpenBrace()
                                .AutoCompleteWith("}", atEndOfSpan: true),
                         new MarkupBlock())),
                 new RazorError(
-                    RazorResources.FormatParseError_Expected_X("}"),
-                    new SourceLocation(14, 0, 14),
+                    RazorResources.FormatParseError_Expected_EndOfBlock_Before_EOF("section", "}", "{"),
```

Comment:
Do we reference the error message in all the tests in this type (rather than handwriting it?)